deploy/runtools/firesim_topology_with_passes.py

```diff
@@ -550,8 +550,10 @@ def screens() -> None:
                     # If AutoILA is enabled, use the following condition
                     if "2 Sockets in" in screenoutput and "hw_server" in screenoutput and "virtual_jtag" in screenoutput:
                         break
-                    # If AutoILA is disabled, use the following condition
-                    elif "No Sockets found" in screenoutput:
```

Comment:
I think you should keep this logic and the new `if` statement turns into:

```
if "No sockets found":
  break
if  "2 sockets...:
  break
elif "fsim" or "switch":
  time.sleep()
  continue
else:
  rootLogger.warning("Unknown screen state. Breaking poll")
  break
```

This makes it clearer that if somethings off it will log it and exit.